test/system/070-build.bats

```diff
@@ -30,6 +30,31 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman build - global runtime flags test" {
+    skip_if_remote "FIXME: pending #7136"
+
+    rand_filename=$(random_string 20)
+    rand_content=$(random_string 50)
+
+    tmpdir=$PODMAN_TMPDIR/build-test
+    run mkdir -p $tmpdir || die "Could not mkdir $tmpdir"
+    containerfile=$tmpdir/Containerfile
+    cat >$containerfile <<EOF
+FROM $IMAGE
+RUN apk add nginx
+RUN echo $rand_content > /$rand_filename
+EOF
+
+    # The 'apk' command can take a long time to fetch files; bump timeout
+    PODMAN_TIMEOUT=240 run_podman --runtime-flag debug build -t build_test --format=docker $tmpdir
+    is "$output" ".*logging has already been configured" "runtime debug seen in log"
```

Comment:
Is `logging has already been configured` a result of `apk`, or of podman? I would strongly encourage you to not use `apk`: it is guaranteed to cause flakes.

Reply:
The test is expecting the DEBUG, but not shown from the actual system test output, is this the right way to test it? It can pass if I run the tests locally
```
$ podman --runtime-flag debug build .
STEP 1: FROM quay.io/libpod/alpine_labels:latest
STEP 2: RUN apk add nginx
DEBU[0000] nsexec:601 nsexec started                    
DEBU[0000] child process in init()                      
DEBU[0000] logging has already been configured          
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
(1/2) Installing pcre (8.42-r0)
(2/2) Installing nginx (1.14.2-r2)
Executing nginx-1.14.2-r2.pre-install
Executing busybox-1.28.4-r1.trigger
OK: 6 MiB in 15 packages
--> bcd76f59f22
STEP 3: RUN echo rand_content > rand_filename
DEBU[0000] nsexec:601 nsexec started                    
DEBU[0000] child process in init()                      
DEBU[0000] logging has already been configured          
STEP 4: COMMIT
--> 2f92146a94a
2f92146a94ab70613822e2570bdc324a81716c9881e899adf432692ab58ec8fa
```

Reply:
Try just this for a test
```bash
    cat >$dockerfile ... <<EOF
FROM $IMAGE
RUN echo $rand_content
EOF

   run_podman build ...
    is "$output" ".*logging has already been configured" "build output includes debug messages"
   is "$output" ".*$rand_content" "build output includes expected content"
```